### PR TITLE
Final blog entry for Webots 2018a

### PR DESCRIPTION
**Pull request by tn12787**
ditto.

### CODE REVIEW

**Comment by tn12787** [comment]
👍 ready for a review.

**Comment by fabienrohrer** [comment]
In viewer: https://www.cyberbotics.com/doc/blog/Webots-2018-a-release?version=update-blog-post-2018a

**Comment by omichel** [comment]
:eyes:

File: blog/Webots-2018-a-release.md [hunk 1]
```diff
@@ -1,10 +1,12 @@
...
+Today we're happy to announce the release of the all-new Webots 2018a, packed with some great new features. We're going to talk about some of them here, but for a comprehensive list of changes please refer to the ChangeLog, found [here](https://www.cyberbotics.com/dvd/common/doc/webots/ChangeLog.html).
+
+This release brings a change to the way Webots is released. We have been looking for a means of improving our release schedule to provide reliability for our users, and a more strict release schedule that ensures updates are shipped predictably and reliably. Thus, we are migrating from a Major-Minor-Maintenance versioning system to follow the MathWorks model. We will now be aiming to release new editions of Webots annualy, with major intermediate releases every six months, starting with Webots 2018a, then Webots 2018b, then Webots 2019a etc. Maintenance revisions will still be shipped to fix critical bugs between these release milestones.
```
Comment (omichel) [inline blog/Webots-2018-a-release.md]: 2018a &rarr; R2018a (same for R2018b and R2019a). Also I would not mention MathWorks, but rather explain it. For example: "Thus, we are migrating to from Major-Minor-Maintenance to a Year-Release-Mainteance versioning system."

File: blog/Webots-2018-a-release.md [hunk 2]
```diff
@@ -20,9 +22,39 @@ Take better screenshots and record better videos of your robot by aligning the v
...
+
+---
+
+## HTML5 Robot Windows
+
+%figure "HTML5 Robot Windows In Webots"
+![html5 windows](images/html_robot_window.png)
+%end
+
+Over the last couple of years we have been slowly phasing out our native QT-based robot window system, in order to use Web technologies for more powerful layout & design tools. The biggest and most comprehensive step in this task is implementing the default generic robot window in HTML5/JavaScript/CSS, as most of our distributed robot models use the generic window. This update brings a new, shiny, faster generic HTML5-based dockable robot window for use with any robot you create:
```
Comment (omichel) [inline blog/Webots-2018-a-release.md]: Shouldn't it be spelled "Qt-based" instead of "QT-based"?

File: blog/Webots-2018-a-release.md [hunk 3]
```diff
@@ -43,3 +75,29 @@ Furthermore, a model of a motorbike and a model of a scooter have been added:
...
+
+This dialog lets you tune specific parameters related to the X3DOM scene rendered in the browser.
+
+---
+
+## Extra Goodies
+
+Movies can now be recorded in slow motion.
+
+To ensure reproducable simulations, floating-point precision when saving files has been maximised.
```
Comment (omichel) [inline blog/Webots-2018-a-release.md]: Shouldn't it be "reproducible" instead of "reproducable"?

**Comment by tn12787** [comment]
👍

**Pull request merged**